### PR TITLE
rpi-eeprom-config: Improve robustness of padding code

### PR DESCRIPTION
**Pull request by timg236**
Make it obvious that the modifiable files are 4K aligned regions in the EEPROM and add support for files with different padding (for future use).

### CODE REVIEW

**Comment by timg236** [comment]
@peterharperuk Some preliminary changes that fell out of the custom CA cert investigations.

File: rpi-eeprom-config [hunk 1]
```diff
@@ -39,6 +37,11 @@ FILE_HDR_LEN = 20
 FILENAME_LEN = 12
 TEMP_DIR = None
 
+# Modifaiable files are stored in a single 4K erasable sector.
```
Comment (peterharperuk) [inline rpi-eeprom-config:40]: Modifaiable => Modifiable

**Comment by peterharperuk** [comment]
Can't spot any issues

File: rpi-eeprom-config (hunk 1 above)
Comment (timg236) [reply rpi-eeprom-config]: Fixed

**Pull request merged**